collector/README.md

```diff
@@ -363,6 +363,13 @@ The mandatory `<PROFILER>` argument must be one of the following.
     [`massif-visualizer`](https://github.com/KDE/massif-visualizer); the latter
     is recommended, though it sometimes fails to read output files that
     `ms_print` can handle.
+- `bytehound`: Profile with
+  [Bytehound](https://github.com/koute/bytehound), a memory profiler. You must have
+  `libbytehound.so` available in your `LD_LIBRARY_PATH` for it to work.
```

Comment:
I don't think so, because I don't know it's path. The comment should say "you have to add the directory containing `libbytehound.so` to `LD_LIBRARY_PATH`".

This is the same as with other profilers, e.g. to use `Massif`, you must have `valgrind` available in your `PATH`, otherwise it will not work. I just wanted to call it out here, because instead of putting a binary directory into `PATH`, you have to put a library directory into `LD_LIBRARY_PATH`, so it's a bit special.